### PR TITLE
Align output format

### PR DESCRIPTION
**Pull request by ypark234**
Align output format with set width of 15.

### CODE REVIEW

File: random_walk.cpp [hunk 1]
```diff
@@ -5,12 +5,14 @@
 #include <fstream>
 #include <iomanip>
 #include <random>
+using namespace std;
```
Comment (aaswenson) [inline random_walk.cpp]: Are we sure we want to declare std? I don't know the convention on this, I was avoiding it to make everything explicit. If we were to use a different namespace in the future, we could have conflicting functions.

**Review by aaswenson — CHANGES_REQUESTED** [review]
I want to take a look at the file (it's not loading in github). I made a comment about the namespace.

File: random_walk.cpp (hunk 1 above)
Comment (aaswenson) [reply random_walk.cpp]: I looked on stack overflow, there seems to be a pretty strong consensus to not declare this. 

https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice

**Comment by ypark234** [comment]
Removed std declaration.

**Review by aaswenson — APPROVED** [review]

**Pull request merged**